CONTRIBUTING.md

```diff
@@ -109,7 +110,7 @@ described [here](https://doc.rust-lang.org/cargo/reference/overriding-dependenci
 
 In order to run integration tests:
 
-1. Run the temporal server using [docker-compose](https://github.com/temporalio/docker-compose).
+1. Run the temporal server using either the [Temporal CLI's integrated dev server](https://github.com/temporalio/cli#start-the-server) or a [Docker container](https://github.com/temporalio/docker-compose#temporal-server-docker-compose-files)
```

Comment:
I don't know if it's even worth mentioning docker-compose anymore.
You might want to add a note about all of the env vars we added since this was written.